### ISSUE TITLE
Can't see bottom dialog buttons on Android TV

### ISSUE THREAD

**Issue by scjacobi** [magneticchen/Daijishou#117]
This may be very specific to my device, but I am trying out the H96 MAX V58 Android 12 TV Box. This issue affects multiple dialogs, but when I'm in the "Download platforms" dialog for example, I cannot see "Index URI," "Cancel," or "Import" at the bottom.  I think they're there because my mouse icon will change from an arrow to a finger when I hover over where they are supposed to be, but the text is either missing, or identical in color to the background, so I can't see it.  For a while I thought I was stuck because I couldn't figure out how I was supposed to progress, but I eventually randomly clicked in the right area and the frontend was able to proceed.  Not sure if there is some "Android TV aware" setting that can correct these dialog buttons, but it would be really cool if there was.

**Comment by scjacobi** [comment]
![IMG20221106203501](https://user-images.githubusercontent.com/11164378/200209085-72278f62-62aa-4776-a1a9-7e8637bebee1.jpg)

**Comment by savethekiddes** [comment]
I'm sorry, Android TVs are not supported for OEM inconsistency, particulary with SAF. I tried to reproduce the issue on two of my Android 12 devices and on one Android 12 tv box but I still haven't got results.

Beside the fact that Daijishou is meant for handhelds, the lack of (Google) guidelines for TVs OEMs leads to an unbelivable fragmentation of software, making near to impossible to solve issues like this. Sorry.

Still, this isn't an ultimatum. Hope that someone else can figure it out.

**Comment by magneticchen** [comment]
> This may be very specific to my device, but I am trying out the H96 MAX V58 Android 12 TV Box. This issue affects multiple dialogs, but when I'm in the "Download platforms" dialog for example, I cannot see "Index URI," "Cancel," or "Import" at the bottom. I think they're there because my mouse icon will change from an arrow to a finger when I hover over where they are supposed to be, but the text is either missing, or identical in color to the background, so I can't see it. For a while I thought I was stuck because I couldn't figure out how I was supposed to progress, but I eventually randomly clicked in the right area and the frontend was able to proceed. Not sure if there is some "Android TV aware" setting that can correct these dialog buttons, but it would be really cool if there was.

Android TV doesn't support SAF. And also after I upgraded mine Google TV with chromecast. I've encountered the same issue as well. And still hasn't figured it out what is going on. As I put my focus mainly on handheld form factors.

**Comment by scjacobi** [comment]
That makes sense, and I understand.  In all honesty, this issue is the only one preventing a flawless experience.  I have been using Daijishou as my primary launcher and now that I have the libraries all set up, it is otherwise working flawlessly.  I can work around the issue for now, and love using it.  Thank you for all your efforts.

**Comment by tristan-k** [comment]
>  but I eventually randomly clicked in the right area and the frontend was able to proceed. 

@scjacobi Can you explain how you did that. I'm running Daijishou on Dynalink 4K flashed with LineageOS and struggeling with the same issue.

**Comment by scjacobi** [comment]
> @scjacobi Can you explain how you did that. I'm running Daijishou on [Dynalink 4K](https://wiki.lineageos.org/devices/wade/) flashed with LineageOS and struggeling with the same issue.

@tristan-k Yeah, I was moving my mouse along the bottom and I noticed that (on Android 12 at least) the mouse icon would change from an arrow to a click-finger when I hovered near the bottom, indicating that there was _something_ there for me to click on, even though I couldn't see what it was.  I compared that to Daijisho running on my tablet, and I could see where the buttons were laid out in the same dialog, so if I went to roughly the same location on the TV screen and clicked, it was as if I clicked "Okay" or "Save" and I could continue.  The buttons are always invisible, and unfortunately don't always correspond to the same relative locations as on my tablet, but with enough trial and error, I was able to find the right locations and proceed.  But it was the changing of the icon that clued me into any of this in the first place.  I imagine LineageOS follows the same context rules for mice, but don't know for a fact.
